src/Pandell.PackageRunner/Token.cs

```diff
@@ -0,0 +1,10 @@
+/*
+* IMPORTANT: This file is modified during build process. Don't modify it unless build script is changed as well.
```

Comment:
Yes, it does. But as long we use the `snk` from this repo it won't change. Only when we are building with TeamCity it will change when we should use a different key file (and TeamCity won't commit changes).